src/main/antlr4/imports/mysql_idents.g4

```diff
@@ -56,4 +59,4 @@ fragment BACKTICK: '`';
 QUOTED_IDENT: BACKTICK (('``') | ~('`'))+ BACKTICK;
 
 fragment DIGIT: [0-9];
-WS  :   [ \t\n\r]+ -> skip ;
+WS  :   [ \t\n\r]+ -> channel(HIDDEN) ;
```

Comment:
So TokenStream walks off channel tokens as well... but if everything is getting replaced with `___MAXWELL___` (including spaces), does it matter at all?